bankayma_account/data/mail_template_views.xml

```diff
@@ -0,0 +1,77 @@
+            <p style="margin: 0px; padding: 0px; font-size: 13px;">
+                Dear
+                <t t-if="object.partner_id.parent_id">
+                    <t
+                        t-out="object.partner_id.name or ''"
+                        t-translation="off"
+                    >Brandon Freeman</t> (<t
+                        t-out="object.partner_id.parent_id.name or ''"
+                        t-translation="off"
+                    >Azure Interior</t>),
+                </t>
+                <t t-else="">
+                    <t
+                        t-out="object.partner_id.name or ''"
+                        t-translation="off"
+                    >Brandon Freeman</t>,
+                </t>
+                <br /><br />Here is your
+                <t t-if="object.move_type == 'out_invoice'">invoice <span
+                        style="font-weight:bold;"
+                        t-out="object.name or ''"
+                        t-translation="off"
+                    >INV/2021/05/0005</span>
+                </t>
+                <t t-else="">bill <span
+                        style="font-weight:bold;"
+                        t-out="object.name or ''"
+                        t-translation="off"
+                    >INV/2021/05/0005</span>
+                </t>
+                <t t-if="object.invoice_origin">(with reference: <t
+                        t-out="object.invoice_origin or ''"
+                        t-translation="off"
+                    >SUB003</t>)
+                </t>amounting in <span
+                    style="font-weight:bold;"
+                    t-out="object.currency_id.format(object.amount_total) or ''"
+                    t-translation="off"
+                >$ 143,750.00</span>
+                from <t
+                    t-out="object.company_id.name or ''"
+                    t-translation="off"
+                >YourCompany</t>.
+                <t
+                    t-if="object.payment_state in ('paid', 'in_payment')"
+                >This invoice is already paid.</t>
+                <t t-else="">
+                    <t t-if="object.move_type == 'out_invoice'">
+                        Please remit payment at your earliest convenience.
+                        <t t-if="object.payment_reference">
+                            <br /><br
+                            />Please use the following communication for your payment: <span
+                                style="font-weight:bold;"
+                                t-out="object.payment_reference or ''"
+                                t-translation="off"
+                            >INV/2021/05/0005</span>.
+                        </t>
+                    </t>
+                    <t t-else="">Payment will be remitted in next round</t>
+                    <br /><br />Do not hesitate to contact us if you have any questions.
+                    <t t-if="object.invoice_user_id.signature">
```

Comment:
is it intentional that the signature is only shown if the payment state is not paid?

the whole thing will get more readable if you use p tags and move most t-ifs there